definitions/divorce/json/FixedLists/FixedLists-general-email-nonprod.json

```diff
@@ -16,7 +16,7 @@
   {
     "LiveFrom": "08/09/2020",
     "ID": "GeneralEmailPartiesEnum",
-    "ListElementCode": "corespondent",
+    "ListElementCode": "co-respondent",
```

Comment:
Would that require a change in COS? 
![image](https://user-images.githubusercontent.com/16866068/94930374-35edb400-04be-11eb-982f-3bfb3bbb53a8.png)

Reply:
Oh aha actually, done in the second PR nice 😅

Reply:
Yes, it would.
https://github.com/hmcts/div-case-orchestration-service/pull/1047